addons/mail/static/src/core/common/thread.js

```diff
@@ -181,6 +183,14 @@ export class Thread extends Component {
         return this.state.showJumpPresent ? PRESENT_THRESHOLD - 200 : PRESENT_THRESHOLD;
     }
 
+    get preferenceButtonText() {
+        const [, before, inside, after] =
+            _t(
+                "<button>Change your preferences</button> to receive new notifications in your Inbox."
```

Comment:
"[...] in your Inbox" => "[...] in your inbox"
No capital letter, as used in Congratulation text when handling notif in Odoo, or the "You inbox is empty" title.